packages/go_router/example/README.md

```diff
@@ -23,9 +23,14 @@ An example to demonstrate how to navigate using named locations instead of URLs.
 ## [Redirection](https://github.com/flutter/packages/blob/main/packages/go_router/example/lib/redirection.dart)
 `flutter run lib/redirection.dart`
 
-An example to demonstrate how to use redirect to handle a sign-in flow.
+An example to demonstrate how to use redirect to handle a simple sign-in flow.
```

Comment:
I'd avoid "simple" as it can be discouraging for folks that don't find the sample simple 😄 

```suggestion
An example to demonstrate how to use redirect to handle a sign-in flow.
```